### ISSUE TITLE
Creating ambiq-rs organization

### ISSUE THREAD

**Issue by felipebalbi** [trembel/ambiq-apollo3-pac#1]
Hi,

I have a redboard from sparkfun and would really like to play with running rust on it. I came to your repository accidentally after generating my own pac with the svd files available from ambiq's SDK. 

I noticed, as well, that the pac could use some update work, perhaps just regenerating it with a newer svd2rust would suffice and I was going to send a PR, but I thought if it would be a nice idea to create ambiq-rs organization and give ownership of your repos to the new org. Then we could have a single repository adding pacs for the entire Apollo line. 

How would you feel about that?

My end goal is try to get support for apollo3 on embassy. 

Cheers

**Comment by gauteh** [comment]
You might be interested in: https://github.com/gauteh/ambiq-rs

**Comment by gauteh** [comment]
https://github.com/rgcottrell/ambiq-apollo3-pac/issues/2

**Comment by trembel** [comment]
As I really have no time to work on the Ambiq, I'm very happy to hand over the maintenance and ownership of this to any of you (also of the crates repo).. Let me know how to handle this..

**Comment by trembel** [comment]
And I would really appeciate an embassy implementation for it :) (especially for the apollo 4 though..)

**Comment by gauteh** [comment]
Are you aware of any evaluation boards for the 4? (I had to create apollo-pac2 a long time ago: https://github.com/gauteh/ambiq-rs/tree/main/pac/apollo3, but it can hopefully be replaced by an updated version of this or the other).

**Comment by trembel** [comment]
> Are you aware of any evaluation boards for the 4? (I had to create apollo-pac2 a long time ago: https://github.com/gauteh/ambiq-rs/tree/main/pac/apollo3, but it can hopefully be replaced by an updated version of this or the other).

No, not really. I was lucky to get an official one over my company.
